scripts/ipfs/ipfs-swarm-cjdns.sh

```diff
@@ -30,7 +30,7 @@ while read -r cjdns_peer; do
     # XXX: The below command hasn't been working -- so for now only 1-hop peers are checked
     #peers+=$(cjdnstool query getpeers $peer | sed -e '1d;$d' |awk -F. '{ print $6".k" }')
 
-done <<< `sudo nodejs /opt/cjdns/tools/peerStats 2>/dev/null | awk '{ if ($2 == "ESTABLISHED") print $1 }' | awk -F. '{ print $6".k" }' | xargs`
+done <<< `sudo nodejs /opt/cjdns/tools/peerStats 2>/dev/null | awk '{ if ($3 == "ESTABLISHED") print $2 }' | awk -F. '{ print $6".k" }' | xargs`
```

Comment:
Seeing things like this, I wonder how yrd will be affected. It should be tested before merging, it might not be valid anymore, or it might need a rebuild. It's last commit to master was on April 20th.

Reply:
It depends how yrd does things. If its direct to the api you may be fine as peerStats polls the api and the returns values.

I'm not familiar with yrd nor have any idea how to actually use it:)

Reply:
I wonder if cjdns itself will soon have a `cjdnsctl` type of program bundled in. In the meantime yrd is the best candidate. I think this is reasonable to make a core dependency. @darkdrgn2k thoughts?

Reply:
peerStats uses a api wrapper for node js

I know CJD has got some new command line tools working. For one he has a working traceroute finally :) Should get looked into.But its still node.js from what i understand

Only thing that i find odd about yrd is the service that needs to run, why would a "ifconfig" like app need a service?!?!

Reply:
>Only thing that i find odd about yrd is the service that needs to run, why would a "ifconfig" like app need a service?!?!

Yeah idk. I think it runs ok without it, :question: